src/firefly/js/util/Color.js

```diff
@@ -32,13 +32,13 @@ const colours = {
 
 
 function toRGBA(/* String */ color, /* Number */ alpha) {
-    var num = parseInt(color, 16);
+    const num = parseInt(color[0]==='#' ? color.substr(1) : color, 16);
     return [num >> 16, num >> 8 & 255, num & 255, alpha];
 }
 
 function toRGB(/* String */ color) {
-        var num = parseInt(color, 16);
-        return [num >> 16, num >> 8 & 255, num & 255];
+    const  num = parseInt(color[0]==='#' ? color.substr(1) : color, 16);
+    return [num >> 16, num >> 8 & 255, num & 255];
 }
 
```

Comment:
you may also call function getRGBA() in Color.js which handles the color string like '#...', 'rgb(...)', 'rgba(...)' or a pure color name.